packages/vscode-rocket/src/extension.ts

```diff
@@ -18,6 +18,7 @@ export async function activate(context: vscode.ExtensionContext) {
   const workspaceRoot = path.resolve(rootDir);
 
   // @ts-ignore
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
This was flagged as an ESLint warning. I thought this was the initial cause fo the failed build, so I added an ESLint ignore directive. Happy to remove this

Reply:
no worries - this vscode extension is still just a POC